astropy/io/fits/column.py

```diff
@@ -461,7 +461,9 @@ def __init__(self, name=None, format=None, unit=None, null=None,
                  array=None, ascii=None):
         """
         Construct a `Column` by specifying attributes.  All attributes
-        except `format` can be optional.
+        except format can be optional, see :ref:`column_creation` and
+        :ref:`creating_ascii_table` for more information regarding the
+        ``TFORM`` keyword.
```

Comment:
Instead of just "regarding the TFORM keyword" change this to "regarding TFORMn keywords"--there is no single TFORM keyword, but really a whole class of keyword that take the format TFORMn, where n is a positive integer.  This is consistent with how these are typically spelled when discussing FITS.